### PR TITLE
i18n(fr): Add `guides/pages`

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description
Add `guides/pages` in french language.

- Related tag : i18n

### CODE REVIEW

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+- src/
+  - content/
+    - docs/
+      - hello-world.md
+      - reference/
+        - faq.md
+
+</FileTree>
+
+### Sécurité de la typologie du frontmatter
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:39]:
```suggestion
### Frontmatter avec sûreté du typage
```

We usually use "typage" ([reference](https://www.larousse.fr/dictionnaires/francais/typage/186838)) as "typologie" refers to an entirely different domain.

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+Toutes les pages Starlight partagent un [ensemble commun de propriétés de la page d'accueil](/fr/reference/frontmatter/) personnalisable qui permet de contrôler l'apparence de la page :
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:41]:
```suggestion
Toutes les pages Starlight partagent un [ensemble commun de propriétés du frontmatter](/fr/reference/frontmatter/) personnalisable qui permet de contrôler l'apparence de la page :
```

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+```md
+---
+title: Bonjour tout le monde !
+description: Voici une page de mon site alimenté par Starlight
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:46]:
```suggestion
description: Voici une page de mon site propulsé par Starlight
```

I think the [original translation](https://starlight.astro.build/fr/getting-started/#frontmatter-avec-s%C3%BBret%C3%A9-du-typage) was a bit less weird even tho "powered" usually lead to weird translations in French.

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+---
+```
+
+Si vous oubliez quelque chose d'important, Starlight vous le fera savoir.
+
+## Pages personnalisées
+
+Pour les cas d'utilisation avancés, vous pouvez ajouter des pages personnalisées en créant un répertoire `src/pages/`.
+Le répertoire `src/pages/` utilise le [routage basé sur les fichiers d'Astro](https://docs.astro.build/fr/basics/astro-pages/#file-based-routing) et inclut le support des fichiers `.astro` parmi d'autres formats de pages.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:55]:
```suggestion
Le répertoire `src/pages/` utilise le [routage basé sur les fichiers d'Astro](https://docs.astro.build/fr/basics/astro-pages/#routage-bas%C3%A9-sur-les-fichiers) et inclut le support des fichiers `.astro` parmi d'autres formats de pages.
```

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+  - content/
+    - docs/
+      - hello-world.md
+  - pages/
+    - custom.astro
+    - archived.html
+
+</FileTree>
+
+Pour en savoir plus, consultez le guide ["Pages" dans la documentation Astro](https://docs.astro.build/fr/basics/astro-pages/).
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:72]:
```suggestion
Pour en savoir plus, consultez le guide [« Pages » dans la documentation d'Astro](https://docs.astro.build/fr/basics/astro-pages/).
```

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+
+### Utiliser le design de Starlight dans des pages personnalisées
+
+Pour utiliser la mise en page Starlight dans des pages personnalisées, englobez le contenu de votre page avec le composant `<StarlightPage />`.
+Cela peut s'avérer utile si vous générez du contenu de manière dynamique, mais que vous souhaitez tout de même utiliser la conception de Starlight.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:77]:
```suggestion
Cela peut s'avérer utile si vous générez du contenu de manière dynamique, mais que vous souhaitez tout de même utiliser le design de Starlight.
```

I think we should keep the consistency with the heading like the English version and "conception" does not feel right to me here.

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+
+```astro
+---
+// src/pages/custom-page/example.astro
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:81]:
```suggestion
// src/pages/page-perso/exemple.astro
```

We usually try to translate French filenames.

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+
+<StarlightPage frontmatter={{ title: 'Ma page personnalisée' }}>
+	<p>Il s'agit d'une page personnalisée avec un composant personnalisé :</p>
+	<CustomComponent />
+</StarlightPage>
+```
+
+#### Props
+
+Le composant `<StarlightPage />` accepte les éléments suivants.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:94]:
```suggestion
Le composant `<StarlightPage />` accepte les props suivantes.
```

We lost the correct translation between the previous line and this one.

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 9]
```diff
@@ -0,0 +1,151 @@
...
+
+##### `frontmatter` (required)
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:96]:
```suggestion
##### `frontmatter` (obligatoire)
```

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 10]
```diff
@@ -0,0 +1,151 @@
...
+
+**type :** `StarlightPageFrontmatter`
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:98]:
```suggestion
**Type :** `StarlightPageFrontmatter`
```

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 11]
```diff
@@ -0,0 +1,151 @@
...
+
+Définit les [propriétés de la page d'accueil](/fr/reference/frontmatter/) pour cette page, similaire à la page d'accueil dans les pages Markdown.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:100]:
```suggestion
Définit les [propriétés du frontmatter](/fr/reference/frontmatter/) pour cette page, similaire au frontmatter dans les pages Markdown.
```

Any specific reason you sometime keep "frontmatter" (as expected) and sometime you translate it to "page d'accueil"?

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 12]
```diff
@@ -0,0 +1,151 @@
...
+
+{/* **Type :** `SidebarEntry[] | undefined` */}
+{/* **Par défaut :** la barre latérale générée en fonction de la [configuration globale `sidebar`](/fr/reference/configuration/#sidebar) */}
+
+{/* Fournir une barre latérale de navigation personnalisée pour cette page. */}
+{/* Si elle n'est pas définie, la page utilisera la barre latérale globale par défaut. */}
+
+##### `hasSidebar`
+
+**Type :** `boolean`
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:119]:
```suggestion
**Type :** `boolean`  
```

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 13]
```diff
@@ -0,0 +1,151 @@
...
+**Par défaut :** `false` si [`frontmatter.template`](/fr/reference/frontmatter/#template) est `'splash'`, autrement `true`
+
+Contrôle l'affichage ou non de la barre latérale sur cette page.
+
+##### `headings`
+
+**Type :** `{ depth: number; slug: string; text: string }[]`
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:126]:
```suggestion
**Type :** `{ depth: number; slug: string; text: string }[]`  
```

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 14]
```diff
@@ -0,0 +1,151 @@
...
+**Par défaut :** `[]`
+
+Fournir un tableau de tous les titres de cette page.
+Starlight générera la table des matières de la page à partir de ces titres s'ils sont fournis.
+
+##### `dir`
+
+**Type :** `'ltr' | 'rtl'`
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:134]:
```suggestion
**Type :** `'ltr' | 'rtl'`  
```

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 15]
```diff
@@ -0,0 +1,151 @@
...
+**Par défaut :** `false` si [`frontmatter.template`](/fr/reference/frontmatter/#template) est `'splash'`, autrement `true`
+
+Contrôle l'affichage ou non de la barre latérale sur cette page.
+
+##### `headings`
+
+**Type :** `{ depth: number; slug: string; text: string }[]`
+**Par défaut :** `[]`
+
+Fournir un tableau de tous les titres de cette page.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:129]:
```suggestion
Fournit un tableau de tous les titres de cette page.
```

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 16]
```diff
@@ -0,0 +1,151 @@
...
+Starlight générera la table des matières de la page à partir de ces titres s'ils sont fournis.
+
+##### `dir`
+
+**Type :** `'ltr' | 'rtl'`
+**Par défaut :** le sens d'écriture pour la locale actuelle
+
+Set the writing direction for this page’s content.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:137]:
```suggestion
Définit le sens d’écriture pour le contenu de la page.
```

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 17]
```diff
@@ -0,0 +1,151 @@
...
+
+##### `lang`
+
+**Type :** `string`
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:141]:
```suggestion
**Type :** `string`  
```

File: docs/src/content/docs/fr/guides/pages.mdx [hunk 18]
```diff
@@ -0,0 +1,151 @@
...
+**Par défaut :** la langue de la locale actuelle
+
+Définir la balise de langue BCP-47 pour le contenu de cette page, par exemple `en`, `zh-CN`, ou `pt-BR`.
+
+##### `isFallback`
+
+**Type :** `boolean`
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/pages.mdx:148]:
```suggestion
**Type :** `boolean`  
```

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the new page translation :raised_hands:

I left a few suggestions and comments but overall it looks pretty solid. :thumbsup:

File: docs/src/content/docs/fr/guides/pages.mdx (hunk 1 above)
Comment (thomasbnt) [reply docs/src/content/docs/fr/guides/pages.mdx:39]: Oh great!

File: docs/src/content/docs/fr/guides/pages.mdx (hunk 11 above)
Comment (thomasbnt) [reply docs/src/content/docs/fr/guides/pages.mdx:100]: No, error for my part. I keep frontmatter, I just forgot to edit this part 😶

**Comment by thomasbnt** [comment]
Hello and thanks a lot @HiDeoo for the review!

**Review by HiDeoo — APPROVED** [review]
Thanks for the update, this looks perfect to me :tada:

**Pull request merged**